doc/semieunit.xml

```diff
@@ -0,0 +1,292 @@
+############################################################################
+##
+#W  semieunit.xml
+#Y  Copyright (C) 2016                                   Christopher Russell
+##
+##  Licensing information can be found in the README file of this package.
+##
+#############################################################################
+##
+
+<#GAPDoc Label="IsMcAlisterTripleSemigroup">
+  <ManSection>
+    <Prop Name = "IsMcAlisterTripleSemigroup" Arg = "S"/>
+    <Returns><K>true</K> or <K>false</K>.</Returns>
+    <Description>
+      A <E>McAlister triple semigroup</E> is an E-unitary inverse semigroup
+      <Ref Oper= "IsEUnitaryInverseSemigroup" BookName= "Semigroups" />
```

Comment:
Get rid of the book name bit here. You don't need to do it when you're referring to stuff in the current package. If you've got `BookName = "Semigroups"` elsewhere then get rid of it too.